src/commands/command-fns/thanks.js

```diff
@@ -27,6 +64,23 @@ async function thanks(message) {
       `You have to use the word "for" when thanking someone. ${example}`,
     )
   }
+
+  thankedMembers.forEach(thankedMember => {
+    if (thanksHistory[thankedMember.id]) {
+      thanksHistory[thankedMember.id] += 1
+    } else {
+      thanksHistory[thankedMember.id] = 1
+    }
```

Comment:
Actually, why don't we store a little more info in the history here:

```suggestion
    thanksHistory[thankedMember.id] = thanksHistory[thankedMember.id] ?? []
    thanksHistory[thankedMember.id].push(messageLink)
```

You'll need to move up the `const messageLink = getMessageLink(message)` line that's currently below.

And we'll need to adjust how we retrieve the count for the rankings (just get `.length`).